genie/process_functions.py

```diff
@@ -517,6 +517,9 @@ def _append_rows(new_datasetdf, databasedf, checkby):
         logger.info("No new rows")
     del appenddf[checkby]
     appenddf.reset_index(drop=True, inplace=True)
+    # update the ROW_ID and ROW_VERSION columns to be NaN since they are not in the original database
+    appenddf["ROW_ID"] = np.nan
```

Comment:
What issue occurs when these aren't set to `np.nan` ? This is only happening for certain table scenarios?